src/translations/fr.json

```diff
@@ -382,7 +382,7 @@
     "Substation": "Site",
     "BusBarSections": "Sections de jeu de barre",
     "BusBarSectionID": "ID",
-    "BusBarSectionName": "Nom",
+    "BusBarSectionName": "Nom (optionnel)",
```

Comment:
Maybe check if you could use `NameOptional` key and remove `BusBarSectionName` because it seems like it is the same string :)

Reply:
It's fixed.